File: Java/프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        String str = my_string.replaceAll("[a-zA-Z]", "");
        
        
        for(int i=0; i<str.length(); i++){
            char a = str.charAt(i);
            answer += a - '0';
        }
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.js
function solution(my_string) {
    var answer = 0;
    my_string.toLowerCase();
    
    for(let i=0; i<my_string.length; i++){
        if(my_string[i] == 1 || my_string[i] == 2 || my_string[i] == 3 || 
           my_string[i] == 4 || my_string[i] == 5 || my_string[i] == 6 || 
           my_string[i] == 7 || my_string[i] == 8 || my_string[i] == 9)
          answer += parseInt(my_string[i]);
        } 
    return answer;
}